INSTALL

```diff
@@ -262,27 +265,55 @@ interactive dialog.
 
 Now on with the build:
 
-  make
-  make install
+  make -jX
+
+where X is the number of available cores. Without the -j option
+it may take a little while to build depending on your platform.
+
+Then you can directly run from the build directory:
 
-It may take a little while to build depending on your platform.
+  ./output/bin/qgis
+
+Another option is to install to your system:
+
+  make install
 
 After that you can try to run QGIS:
 
   $HOME/apps/bin/qgis
 
 If all has worked properly the QGIS application should start up and appear
-on your screen.  If you get the error message "error while loading shared libraries",
+on your screen. If you get the error message "error while loading shared libraries",
 execute this command in your shell.
 
   export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:${HOME}/apps/lib/
 
+Optionally, if you already know what aspects you want in your custom build
+then you can skip the interactive ccmake .. part by using the cmake -D
+option for each aspect, e.g.:
+
+  cmake -D CMAKE_BUILD_TYPE=Debug -D CMAKE_INSTALL_PREFIX=${HOME}/apps ..
+
+Also, if you want to speed your build times, you can easily do it with ninja,
+an alternative to make with similar build options.
+
+For example, to configure your build you can do either one of:
+
+  ccmake -GNinja ..
+
+  cmake -G Ninja -D CMAKE_BUILD_TYPE=Debug -D CMAKE_INSTALL_PREFIX=${HOME}/apps ..
```

Comment:
it works as well, ``ccmake -G Ninja ..`` also works as well. Should I make it consistent? (I'm leaning towards ``-G Ninja``...